app/controllers/api/skater_profiles_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::SkaterProfilesController < ApplicationController
+  def show
+    profile = SkaterProfile.find_by(skater_id: current_skater.id)
+
+    render json: profile
+  end
+
+  def create
+    profile = SkaterProfile.new(create_params)
+    profile.save!
+
+    render json: profile
+  end
+
+  def update
+    profile = SkaterProfile.find_by(skater_id: current_skater.id)
+    profile.attributes = update_params
+    profile.save!
```

Comment:
Instead of calling attributes= and save! you can just call update_attributes!

profile.update_attributes!(update_params)